README.md

```diff
@@ -1,63 +1,198 @@
+```
+zappa undeploy dev
 ```
+
+**NOTE**. The `deploy` command always returns a brand new `URL`. The `update`
+command does not change the current `URL`. The `undeploy` command destroys the
+old `URL` forever.
+
+### Threat Response Module
+
+Now, the only things left to do is to follow one of these URLs to navigate 
+to Threat Response in your region and create the Cyberprotect module using your Lambda's URL:
+- US: https://visibility.amp.cisco.com/settings/modules/available/e16a5446-d4ea-44df-8dca-58fd9c7f476f/new
+- EU: https://visibility.eu.amp.cisco.com/settings/modules/available/5be9df3f-40b7-47af-bd16-6b480fc916ba/new
+- APJC: https://visibility.apjc.amp.cisco.com/settings/modules/available/8758d0bf-3c22-4ca3-bc2b-c55fbfeeca44/new
+
+That is it! Your Serverless Relay is ready to use! Congratulations!
+
+## Step 3: Testing (Optional)
+
+If you want to test the application you have to install a couple of extra
+dependencies from the [test-requirements.txt](test-requirements.txt) file:
+```
+pip install --upgrade --requirement test-requirements.txt
+```
+
+You can perform two kinds of testing:
+
+- Run static code analysis checking for any semantic discrepancies and
+[PEP 8](https://www.python.org/dev/peps/pep-0008/) compliance:
+
+  `flake8 .`
+
+- Run the suite of unit tests and measure the code coverage:
+
+  `coverage run --source api/ -m pytest --verbose tests/unit/ && coverage report`
+
+If you want to test the live Lambda you may use any HTTP client (e.g. Postman),
+just make sure to send requests to your Lambda's `URL`.
+
+**NOTE.** If you need input data for testing purposes you can use data from the
+[observables.json](observables.json) file.
+
+## Implementation Details
+
+### Implemented Relay Endpoints
+
+- `POST /health`
+  - Checks that the service is available at the moment.
+
+- `POST /deliberate/observables`
+  - Accepts a list of observables and filters out unsupported ones.
+  - Makes a series of requests to the underlying external service to query for
+  some cyber threat intelligence data on each supported observable.
+  - Maps the fetched data into appropriate CTIM entities.
+  - Returns a list per each of the following CTIM entities (if any extracted):
+    - `Verdict`.
+
+- `POST /observe/observables`
+  - Accepts a list of observables and filters out unsupported ones.
+  - Makes a series of requests to the underlying external service to query for
+  some cyber threat intelligence data on each supported observable.
+  - Maps the fetched data into appropriate CTIM entities.
+  - Returns a list per each of the following CTIM entities (if any extracted):
+    - `Verdict`,
+    - `Judgement`.
+
+### Supported Types of Observables
+
+- `ip`
+- `url`
+
+### Supported Environment Variables
+CyberCrime Tracker does not have any Environment Variables.
+
+## CTIM Mapping Specifics
+
+The CyberCrime Tracker API has a simple mapping. It is based on only one value of `message`.
```

Comment:
Could you expand or rephrase `It is based on only one value of message`?

`The CyberCrime Tracker API has a simple mapping and it is completely based on value of `message`. field` ?